### PR TITLE
singupform updated

### PR DESCRIPTION
**Pull request by ghost**
userschema includes joinDate property;
singup form submit creates a new user in the db

### CODE REVIEW

File: server/users/userModel.js [hunk 1]
```diff
@@ -14,7 +14,27 @@ var UserSchema = new mongoose.Schema({
     type: String,
     required: true
   },
-  salt: String
+  first: String,
+  last: String,
+  male: Boolean,
+  female: Boolean,
```
Comment (dduugg) [inline server/users/userModel.js:20]: would it make more sense to have a single "gender" field?

File: server/users/userController.js [hunk 2]
```diff
@@ -29,38 +29,34 @@ module.exports = {
...
   // },
 
-  // signup: function (req, res, next) {
-  //   var username  = req.body.username,
-  //       password  = req.body.password,
-  //       create,
-  //       newUser;
+  signup: function(req, res, next) {
+    var newUser = req.body;
+    //get username
```
Comment (dduugg) [inline server/users/userController.js]: unnecesarry comment

File: server/users/userController.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-//var User = require('./userModel');
-    // Q    = require('q'),
-    // jwt  = require('jwt-simple');
+var User = require('./userModel.js');
```
Comment (dduugg) [inline server/users/userController.js]: no need for '.js'

File: client/app/signup/signup.js [hunk 4]
```diff
@@ -1,21 +1,76 @@
 angular.module('pledgr.signup', [])
 
-.controller('SignupController', function ($scope) {
+.factory('Auth', function($http) {
```
Comment (dduugg) [inline client/app/signup/signup.js]: factories should probably go into their own folder so they can be used by other modules. see client/app/services in the shortly-angular repo (but these should probably go into client/app/factories)

File: server/users/userController.js (hunk 2 above)
Comment (ghost) [inline server/users/userController.js]: removed comment

File: server/users/userModel.js (hunk 1 above)
Comment (ghost) [inline server/users/userModel.js:20]: will leave for now; can change if necessary for db/charity logic

File: server/users/userController.js (hunk 3 above)
Comment (ghost) [inline server/users/userController.js]: removed .js extension

**Comment by dduugg** [comment]
Try to squash your PRs into fewer commits, but otherwise looks great. :heart_eyes:

**Pull request merged**